FAQ.md

```diff
@@ -1,3 +1,69 @@
+
+- Process fields to arrive at a concrete payload. An example of this is to pass the certificate_id field in full (see [UVCI Format](#uvci_format)) to a name resolution service which will map this to a resource that can be used for further processing, according to the Member State name resolution rules.
+
+## <a id="mandatory_but_missing"></a>Reconciliation of Mandatory Fields and Missing Data
+
+It can be the case that the data required to populate the DGC JSON schema is simply not present in the  source data when queried. An oft-encountered example of this is with the patient name, where often the Family Name (EU Regulations: "Surname") is present, but the Given Name (EU Regulations: "Forename") is not. In the Annex to the proposed  [EU legislation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:52021PC0130) it is stated that Surname(s), Forename(s) shall be given, in that order. In order to make the DGC JSON Schema  as conformant as possible to this legislation, both fields Family Name and Given Name are marked as `["required"]` so we need to supply the values.  However, our data source only has Family Name and not Given Name.
+
+The solution is to use the [DICOM](https://www.dicomstandard.org/) approach of
+[Type 2](http://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_7.4.3) values. In summary, this  allows a value to be specified as mandatory but can be marked as being empty and have no value. The rich [DICOM](https://www.dicomstandard.org/) metadata approach to values is not available to us in JSON (although we could  get very close using XML), but we can approximate the [DICOM](https://www.dicomstandard.org/) approach by setting the mandatory field to an empty value (i.e. empty string for string types, 0 for numeric types). In this way, we can both support a mandatory value but allow the flexibility of handling the case when the source data simply does not
+have the value available for us.
+
+Please note that if the value is available, then it is required to be set, as per  [DICOM Type 2](http://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_7.4.3):
+
+> [... defines] Type 2 Data Elements that shall be included and are mandatory Data Elements. However, it is permissible
+> that if a Value for a Type 2 element is unknown it can be encoded with zero Value Length and no Value.
+> **If the Value is known the Value Field shall contain that value [...].
+> These Data Elements shall be included in the Data Set and their absence is a protocol violation.**
+
+## <a id="uvci_format"></a>Annex 2 specifies three different formats for certificate id (UVCI). Which one do I use?
+
+For _generating_ the certificate id: the choice of which option to choose from Annex 2 in the  [eHealthNetwork Vaccination Interoperability Guidelines](https://ec.europa.eu/health/sites/health/files/ehealth/docs/vaccination-proof_interoperability-guidelines_en.pdf) is delegated to the Member State.
```

Comment:
No examples of UVCI:s?

Did I miss the discussion about which characters that should be included for Luhn mod N?

Reply:
No, not missed. I'm waiting / blocked by this review so I can add the Luhn-Mod-N text + example (@jschlyter really wanted the text in the wiki for some reason so now I can't edit the wiki until this reviewed, approved and merged)